harness/config/commands.yml

```diff
@@ -186,6 +186,10 @@ command('recompile'):
     DOCKER_BUILDKIT: "1"
   exec: |
     #!bash(workspace:/)|@
+    if docker-compose ps app | grep -q 'exited'; then
+      echo "rebuilding app container" >&2
+      passthru docker-compose up -d --build app
```

Comment:
if rebuild is going to rebuild, do you really need the container to be rebuilt? (aka the --build flag)

Reply:
Good question - I was using a `panic()` to test this, with the assumption that the app has exited due to a coding issue, that you might have since fixed and now require the app container to be rebuilt. Without this flag, the previous build of the app container would start and might just die again before it can continue with the recompile process

Reply:
although ... you don't need to recompile if --build :)